cabby/data/metagraph/utils.py

```diff
@@ -103,11 +108,88 @@ def update_osm_map(osm_map: Map,
   print("In update_osm_map: %d nodes in graph after adding." % (
     osm_map.nx_graph.number_of_nodes()))
 
+
+def add_conceptual_nodes_and_edges(graph: nx.Graph,
+                                   poi_map: Dict[Text, int],
```

Comment:
We've used the Text type previously in the code but are switching it to 'str' for new code and old code we touch (as per typing conventions with py3). Can you change it to str here and elsewhere in new code?

Reply:
Fixed by d72eab9effdde39d8640de925346ec1c8bb90b3c